### PR TITLE
Release notes week of 5-5-2015

### PR DESCRIPTION
**Pull request by srpearce**
@lamagnifica @explorerleslie @griffresch @cptvitamin Please review and let me know if there's anything you'd like me to add or delete.

### CODE REVIEW

File: en_us/release_notes/source/lms/index.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 edX Learning Management System
 ####################################
 
+*********************
+May 5, 2015
+*********************
+
+* A problem in which some students could not select **Change Email** to change
```
Comment (lamagnifica) [inline en_us/release_notes/source/lms/index.rst]: learners?

**Comment by lamagnifica** [comment]
:+1: nothing from Victor & Analytics, and "nothing for mobile at this time" per Lou

**Comment by explorerleslie** [comment]
@catong could you work with @srpearce to add a teaser to "Coming soon" about the new grade reports? The info provided in the two tasks (certificate/track info and per-problem info) is highly requested by course teams, so they'll be excited about it.

**Comment by catong** [comment]
Will do!

File: en_us/release_notes/source/coming_soon.rst [hunk 2]
```diff
@@ -15,12 +15,4 @@ A new Student Notes tool allows learners in some courses to highlight and make
 notes about text directly in the courseware. Learners can see individual notes
 in the courseware or all notes together on a new **Notes** tab in the LMS.
```
Comment (catong) [inline en_us/release_notes/source/coming_soon.rst:35]: @sylvia, can you please add this to the Coming Soon file? @explorerleslie how does this look:

Grade report enhancements

Course teams will be able to download a new problem-level grade report from the Data Download tab in the LMS. This new report will show the possible and earned points for each problem in an assignment, the total weighted score for each problem, and each learner's overall grade.

In addition, the existing grade report will be enhanced with information about each learner’s certificate status. New columns will show learners’ enrollment tracks, whether they have completed ID verification, whether they have earned a certificate for the course (based on attaining a passing grade at the time that the grade report is requested), the type of certificate earned, and whether certificates have been generated for the course.

**Comment by explorerleslie** [comment]
@catong @srpearce :+1:

**Pull request merged**